app/src/main/kotlin/compiler/intermediate_form/IntermediateFormTreeNode.kt

```diff
@@ -4,6 +4,10 @@ import compiler.ast.Function
 import compiler.ast.Variable
 
 sealed class IntermediateFormTreeNode {
+    companion object {
+        const val UNIT_VALUE: Long = 0
```

Comment:
Is this convention used in the whole repository (`Unit` being a zero value). If not, then create a GitHub issues to unify this at some point.

Reply:
The only place (at the moment) was `createGraphForEachFunction`, which I changed.

Reply:
OK, turns out that I can't change that because tests for `createGraphForEachFunction` require the possibility of empty CFG for an expression. Nevertheless, at the moment this is the only place assuming anything about representation of unit.